core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.js

```diff
@@ -0,0 +1,691 @@
+const donatePage = '.modal-backdrop.fade';
+const thanksForDonatingPage = '.modal-backdrop.fade.show';
+
+module.exports = class LoggedInUsers extends baseUser {
+  /**
+   * Function for navigating to the home page.
+   */
+  async navigateToHome() {
+    await this.goto(homeUrl);
+  }
+
+  /**
+   * Function for navigating to the about page.
+   */
+  async navigateToAboutPage() {
+    await this.goto(aboutUrl);
+  }
+
+  /**
+   * Function for navigating to the about foundation page.
+   */
+  async navigateToAboutFoundationPage() {
+    await this.goto(aboutFoundationUrl);
+  }
+
+  /**
+   * Function for navigating to the Thanks for Donating page.
+   */
+  async navigateToThanksForDonatingPage() {
+    await this.page.goto(thanksForDonatingUrl, {
+      waitUntil: 'load',
+      timeout: 0
+    });
+  }
+
+  /**
+   * Function to click the About button in the About Menu on navbar
+   * and check if it opens the About page.
+   */
+  async clickAboutInAboutMenuOnNavbar() {
+    await this.clickOn(navbarAboutTab);
+    await this.clickOn(navbarAboutTabAboutButton);
+    if (this.page.url() !== aboutUrl) {
+      throw new Error (
+        'The About Oppia button in About Menu on navbar ' +
+        'does not open the About page!');
+    } else {
+      showMessage(
+        'The About Oppia button in About Menu on navbar ' +
+        'opens the About page.');
+    }
+  }
+
+  /**
+   * Function for click the Browse Our Lessons button in the About page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickBrowseOurLessonsButtonInAbout() {
+    await this.clickOn(browseOurLessonsButton);
+    if (this.page.url() !== mathClassroomUrl) {
+      throw new Error (
+        'The Browse our lesson button does not open the Math Classroom page!');
+    } else {
+      showMessage(
+        'The Browse our lesson button opens the Math Classroom page.');
+    }
+  }
+
+  /**
+   * Function for click the Access Android App button in the About page
+   * and check if it opens the Android page.
+   */
+  async clickAccessAndroidAppButtonInAbout() {
+    await this.clickOn(accessAndroidAppButton);
+    if (this.page.url() !== androidUrl) {
+      throw new Error (
+        'The Access the Android App button does not open the Android page!');
+    } else {
+      showMessage(
+        'The Access the Android App button opens the Android page.');
+    }
+  }
+
+  /**
+   * Function for click the Visit Classroom button in the About page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickVisitClassroomButtonInAbout() {
+    await this.clickOn(visitClassroomButton);
+    if (this.page.url() !== mathClassroomUrl) {
+      throw new Error (
+        'The Visit Classroom button does not open the Math Classroom page!');
+    } else {
+      showMessage(
+        'The Visit Classroom button opens the Math Classroom page.');
+    }
+  }
+
+  /**
+   * Function for click the Browse Our Lessons button in the About page
```

Comment:
Here it's called "browse our lessons" and below it's called "browse library". Does it need to be standardized?